### ISSUE TITLE
Deploy this sim with keyboard navigation and description and new sound/sonification

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#102]
We would like to deploy this sim with keyboard navigation and auditory description. This issue is to track status and progress.

**Comment by jessegreenberg** [comment]
#82 must be resolved before continuing with this issue, but we are likely going to continue with that issue in scenery.

**Comment by jessegreenberg** [comment]
#82 (and its issue in scenery https://github.com/phetsims/scenery/issues/730) has been closed so this issue is no longer on hold.

**Comment by jessegreenberg** [comment]
Now we need to take care of 
- [ ] #100 [keyboard help dialog]
- [ ] #104 [interviews]
- [ ] Any final changes to descriptions in this sim.

**Comment by jessegreenberg** [comment]
It looks like there are changes to descriptions in the Ohms Law design document, adding that to checklist.

**Comment by jessegreenberg** [comment]
We are actually up to date with the design doc.

**Comment by jessegreenberg** [comment]
Interviews and final changes are all that is left.

**Comment by jessegreenberg** [comment]
More changes have been requested for the keyboard shortcuts dialog.

**Comment by emily-phet** [comment]
@jessegreenberg - not waiting for sonification to publish this sim. Interviews are in progress, but don't need to hold anything up. If the descriptions are complete (which it looks like @terracoda has confirmed they are) then this sim can progress towards publication with description.

**Comment by jessegreenberg** [comment]
Sounds good @emily-phet, thanks. We will create an RC for this sim as soon as https://github.com/phetsims/QA/issues/108 is complete.

**Comment by terracoda** [comment]
@jessegreenberg, just mentioning an issue from RIAW https://github.com/phetsims/resistance-in-a-wire/issues/140 here, in case there is a change to the "impurities description".

**Comment by zepumph** [comment]
So https://github.com/phetsims/resistance-in-a-wire/commit/3ee999740a00f10a6d425bb02cde2bc0eab6c83b will need to be pulled into OL as well.

**Comment by terracoda** [comment]
@jessegreenberg and @zepumph, FYI, I made no changes to Ohm's Law descriptions regarding the way the "impurities" description.

**Comment by zepumph** [comment]
Good to know, we were working fast yesterday and didn't really see that there are no content changes in  phetsims/resistance-in-a-wire@3ee9997. @terracoda can you confirm that this sim is ready to go. Removing on hold label.

**Comment by terracoda** [comment]
@zepumph, is there anything keeping Ohm's Law out of an Dev Test or RC Test? Perhaps, we are just waiting on finishing up RIAW, as there are some open issues with that sim?

**Comment by terracoda** [comment]
@zepumph, sorry, I miss-read https://github.com/phetsims/ohms-law/issues/102#issuecomment-383342714. I will confirm this is ready to go today, and remove on-hold label as soon as that is done.

**Comment by terracoda** [comment]
@zepumph, the descriptions of impurities in the wire is good to go.

**Issues I found are:**

- I noticed that keyboard focus is lost, "moves to Sim Resources Region" when I use the ESC key to close the Keyboard Help dialog. Focus is lost only if visual focus is not on the close button, i.e., when system focus is not visible and the virtual cursor is somewhere in the dialog's content. If I hit ESC with focus on the close button, focus is not lost. If I close the dialog with the close button, focus is not lost.
  - Note this issue might have been resolved in RIAW
- The Keyboard Shortcuts button, does not read out as a a "pop-up" button, so I am not sure it is coded as one. It is technically a pop-up button and it should have an aria-haspopup="dialog".
  - this would be a general issue.

I'll create these 2 issues, and then we should be good to go for testing (Dev or RC?) I am not sure where are at with this one.

**Comment by terracoda** [comment]
@zepumph, just confirming that this sim is ready to go.

**Comment by zepumph** [comment]
After talking with @emily-phet today. This sim has sat long enough that we are going to wait for @jbphet to work on tambo sound and sonification for it before releasing. Instead I will focus my efforts on friction. Unassigning.

**Comment by jessegreenberg** [comment]
This is being collapsed into #125. The sim has passed a round of dev testing and all issues are closed or marked as "fixed pending testing". Closing this issue.
